test/boot.js

```diff
@@ -1,10 +1,10 @@
 import 'webcomponents.js/CustomElements';
 import helperFixture from './lib/fixture';
 
-mocha.setup({ timeout: 10000 });
+mocha.setup({ timeout: 10000 }); // eslint-disable-line no-undef
```

Comment:
to avoid the `eslint-disable-line`, you could add `mocha` to the `env` section of `.eslintrc`